xarray/core/computation.py

```diff
@@ -1808,6 +1810,13 @@ def where(cond, x, y):
     Dataset.where, DataArray.where :
         equivalent methods
     """
+    if keep_attrs is None:
+        keep_attrs = _get_keep_attrs(default=False)
+
+    if keep_attrs is True:
+        # keep the attributes of x by default
+        keep_attrs = lambda attrs, context: attrs[1]
```

Comment:
What is the reasoning for making `keep_attrs=True` copy from the first argument?

I believe this is the same as `keep_attrs='override'`, which is arguably more descriptive.

Reply:
it's the second argument, actually. In https://github.com/pydata/xarray/pull/4687#issuecomment-745036433 and the following comments, we thought it was good to keep the attrs of `x` in `xr.where(cond, x, y)` instead of `cond`. I would have been able to just use `duck_array_ops.where_method` instead of `duck_array_ops.where`, but that seems to have other side-effects.

Reply:
Oh, right, that makes sense. I guess this is reasonable then!

Could you add a brief comment to the code explaining this for future readers?

Reply:
I can extend the existing one, sure.